con-master/src/main/java/ba/codecentric/master/controller/HotelController.java

```diff
@@ -2,31 +2,56 @@
 
 import ba.codecentric.base.domain.Hotel;
 import ba.codecentric.base.service.HotelService;
-import org.apache.log4j.Logger;
+import ba.codecentric.base.service.ImageService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestPart;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
 
 import javax.validation.Valid;
+import java.io.IOException;
 
 @RestController
 public class HotelController {
 
     private final HotelService hotelService;
-    private final static Logger LOG = Logger.getLogger(HotelController.class);
```

Comment:
Why did you remove Logger? You should log some events, eg. if hotel is properly saved in db, if image doesn't exist, if image is found and on which path.